src/GDGUkraine/api.py

```diff
@@ -65,9 +65,12 @@ def get_all_gdg_places(session, filtered=False):
     return q.order_by(Place.city).all()
 
 
-def find_event_by_id(session, id):
-    id = int(id)
-    return session.query(Event).get(id)
+def find_event_by_id(session, id_):
+    try:
+        id_ = int(id_)
+        return session.query(Event).get(id_)
+    except ValueError:
+        return None
```

Comment:
I wouldn't promote C-style in Python. You shouldn't mask errors with magical values. It's not worth remembering to do `if` checks upon each call.